public/static/js/subscriptions.js

```diff
@@ -1,52 +1,52 @@
-const SUBSCRIPTION_JIRA_COMMENT = 'jira-comment';
-const SUBSCRIPTION_JIRA_ISSUE = 'jira-issue';
-const SUBSCRIPTION_SLACK = 'slack';
-const SUBSCRIPTION_EMAIL = 'email';
-const SUBSCRIPTION_EVERGREEN_WEBHOOK = 'evergreen-webhook';
+const SUBSCRIPTION_JIRA_COMMENT = "jira-comment";
```

Comment:
Did you mean to format this file? It's fine that you did, just wanted to double check

Reply:
oops, no i didnt!! thank you!